.github/workflows/deploy_prod.yml

```diff
@@ -9,9 +9,7 @@ on:
   pull_request:
     branches:
       - release
-    types:
-      - closed
-  workflow_dispatch: # This enables manual triggers
+    types: [closed, edited] # PR opened, edited(PR title, etc)
 
```

Comment:
workflow_dispatch doesn't work here due to lacking reference to a PR
` if: github.event.pull_request.merged == true && github.event.pull_request.base.ref == 'release'`